### PR TITLE
Rephrased text to remove passive voice for a more active one.

### PR DESCRIPTION
**Pull request by JoelMon**
changelog: none

### CODE REVIEW

File: book/src/continuous_integration/github_actions.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # GitHub Actions
 
-On the GitHub hosted runners, Clippy from the latest stable Rust version comes
-pre-installed. So all you have to do is to run `cargo clippy`.
+The latest stable version of Rust has Cippy pre-installed on GitHub's hosted runners.
```
Comment (flip1995) [inline book/src/continuous_integration/github_actions.md:3]: It is not the latest stable Rust version, that has Clippy pre-installed, it is the GitHub hosted runners that have Clippy pre-installed together with the latest stable Rust version.

The wording can probably still be improved. I wrote that and English is my second language.

**Review by JoelMon — COMMENTED** [review]
Imoved the wording.

File: book/src/continuous_integration/github_actions.md (hunk 1 above)
Comment (Serial-ATA) [inline book/src/continuous_integration/github_actions.md:3]: I'd say it like this:

```suggestion
The latest stable version of Rust provided by Github's hosted runners has Clippy pre-installed.
```

File: book/src/continuous_integration/github_actions.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
...
+It is as simple as running `cargo clippy` to run lints aginst the codebase.
```
Comment (Serial-ATA) [inline book/src/continuous_integration/github_actions.md:4]:
```suggestion
It is as simple as running `cargo clippy` to run lints against the codebase.
```

**Comment by flip1995** [comment]
Thanks! Can you squash your commits please. Then this'll be ready to go.

File: book/src/continuous_integration/github_actions.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
 # GitHub Actions
 
-On the GitHub hosted runners, Clippy from the latest stable Rust version comes
-pre-installed. So all you have to do is to run `cargo clippy`.
+GitHub hosted runners' using the latest stable version of Rust has Clippy pre-installed.
```
Comment (Serial-ATA) [inline book/src/continuous_integration/github_actions.md:3]:
```suggestion
GitHub hosted runners using the latest stable version of Rust have Clippy pre-installed.
```
The apostrophe makes this possessive, there's also a tense mismatch with "using" and "has". :slightly_smiling_face:

Comment (JoelMon) [reply book/src/continuous_integration/github_actions.md:3]: LOL where is my head! Thanks Serial!

**Review by flip1995 — APPROVED** [review]
LGTM. please squash the commits and this is good to go.

**Comment by JoelMon** [comment]
> LGTM. please squash the commits and this is good to go.

I re-based the branch, not sure if I did it right. Let me know. 3af13dbae6619f6e9f9d0dbaec09d567c0473248

**Comment by flip1995** [comment]
Not quite. But I did the rebase+squash for you. Thanks for updating our documentation!

@bors r+

**Pull request merged**